mapo/models/dynamics/tests/test_gaussian.py

```diff
@@ -167,6 +167,14 @@ def test_gaussian_sample(model, state, action):
     assert next_state.dtype == state.dtype
 
 
+@pytest.mark.parametrize("model", get_models())
+def test_gaussian_sample_propagates_gradients(model, state, action):
```

Comment:
So, this is the usual behavior in `tf.distributions`... if the distribution is re-parametrizable then you can indeed backpropagate through the sampling procedure...

It would be nice to add the complementary test here.. to check that once we gradient-stop the sample, trying to get back the gradient will return None (or an error...).

Reply:
Yeah, I'll add it as a "sanity check".